Gems/LmbrCentral/Code/Source/Shape/EditorSphereShapeComponent.h

```diff
@@ -45,13 +53,27 @@ namespace LmbrCentral
     private:
         AZ_DISABLE_COPY_MOVE(EditorSphereShapeComponent)
 
-        // AzFramework::EntityDebugDisplayEventBus
+        // AzFramework::EntityDebugDisplayEventBus overrides ...
         void DisplayEntityViewport(
             const AzFramework::ViewportInfo& viewportInfo,
             AzFramework::DebugDisplayRequests& debugDisplay) override;
 
+        // AzToolsFramework::RadiusManipulatorRequestBus overrides ...
+        float GetRadius() const override;
+        void SetRadius(float radius) override;
+
+        // AzToolsFramework::ShapeManipulatorRequestBus overrides ...
+        AZ::Vector3 GetTranslationOffset() const override;
+        void SetTranslationOffset(const AZ::Vector3& translationOffset) override;
+        AZ::Transform GetManipulatorSpace() const override;
+        AZ::Quaternion GetRotationOffset() const override;
+
         void ConfigurationChanged();
 
-        SphereShape m_sphereShape; ///< Stores underlying sphere representation for this component.
+        SphereShape m_sphereShape; //!< Stores underlying sphere representation for this component.
+
+        using ComponentModeDelegate = AzToolsFramework::ComponentModeFramework::ComponentModeDelegate;
+        ComponentModeDelegate
+            m_componentModeDelegate; //!< Responsible for detecting ComponentMode activation and creating a concrete ComponentMode.
```

Comment:
suggestion: As this line is so long it might be worth moving the comment to the line above so the declaration doesn't need to be split

```
//! Responsible for detecting ComponentMode activation and creating a concrete ComponentMode.
ComponentModeDelegate m_componentModeDelegate;
```

Reply:
Done